setup.py

```diff
@@ -16,10 +16,10 @@
     long_description_content_type="text/markdown",
     url="https://github.com/great-expectations/airflow-provider-great-expectations",
     install_requires=[
-        "apache-airflow>=1.10",
+        "apache-airflow>=2.0",
```

Comment:
A note of caution that would be good for your users to know with this up-leveling of the Airflow version: If users decide to install this version of the provider, their Airflow environment will be upgraded automatically to 2+. I think it would be beneficial to have some sort of note in PyPI and elsewhere that this version of the provider is also only available for Airflow 2 (or Airflow 2.1+ based on the `@apply_defaults` comment above). We can talk more about this tomorrow too!